### PR TITLE
Nightshade web

### PR DESCRIPTION
**Pull request by jburel**
* Remove usage of ``upgrade_webapps`` parameter
* Usage variables

```
PLAY RECAP *******************************************************************************************************************************************************************
ns-web.openmicroscopy.org  : ok=77   changed=6    unreachable=0    failed=0   
```

### CODE REVIEW

File: nightshade-webclients.yml [hunk 1]
```diff
@@ -145,8 +119,13 @@
 
   vars:
     omero_web_config_set: "{{ omero_web_config_set_for_group | combine(omero_web_config_set_for_host) }}"
-
-    nginx_version: 1.14.2
```
Comment (manics) [inline nightshade-webclients.yml]: What's the reason for removing `nginx_version`?

File: nightshade-webclients.yml [hunk 2]
```diff
@@ -145,8 +119,13 @@
...
+    omero_figure_release: "{{ omero_figure_release_override | default('4.0.2') }}"
+    omero_fpbioimage_release: "{{ omero_fpbioimage_release_override | default('0.3.0') }}"
+    omero_iviewer_release: "{{ omero_iviewer_release_override | default('0.7.0') }}"
+    omero_parade_release: "{{ omero_parade_release_override | default('0.1.2') }}"
+    omero_webtagging_autotag_release: "{{ omero_webtagging_autotag_release_override | default('3.0.2') }}"
+    omero_webtagging_tagsearch_release: "{{ omero_webtagging_tagsearch_release_override | default('3.0.3') }}"
+    omero_web_release_version: "{{ omero_web_release_version_override | default('5.4.10') }}"
```
Comment (manics) [inline nightshade-webclients.yml]: Are you planning to upgrade Nightshade OMERO.web to 5.4.10, and the web-apps too?
Current versions on ns-web:
OMERO.py: 5.4.9
```
$ /opt/omero/web/venv/bin/pip list |grep omero
omero-figure (4.0.0)
omero-fpbioimage (0.3.0)
omero-iviewer (0.6.0)
omero-marshal (0.5.3)
omero-parade (0.1.2)
omero-webtagging-autotag (3.0.2)
omero-webtagging-tagsearch (3.0.3)
```

Comment (jburel) [reply nightshade-webclients.yml]: Everything needs to be upgraded. apps and web
If people are ok with change, I will go directly to 5.5.0 for web

File: nightshade-webclients.yml (hunk 1 above)
Comment (jburel) [reply nightshade-webclients.yml]: we did not specify for example in outreach

Comment (manics) [reply nightshade-webclients.yml]: This was introduced during the cyberessentials work as the Nginx verison was out of date: https://github.com/ome/prod-playbooks/pull/150/ . I think it should be brought back so we have full control over the version.

File: nightshade-webclients.yml (hunk 2 above)
Comment (manics) [reply nightshade-webclients.yml]: I think it's important that when a PR is merged it represents the current state of our systems, so depending on your preference either change the versions to match what's currently deployed, or include the 5.5.0 upgrade in this PR.

File: nightshade-webclients.yml (hunk 1 above)
Comment (jburel) [reply nightshade-webclients.yml]: understood

File: nightshade-webclients.yml (hunk 2 above)
Comment (jburel) [reply nightshade-webclients.yml]: I have only run in ``checkmode`` so nothing has been upgraded. 
In my view, It is preferable to have any changes reviewed before running without ``checkmode`` if possible

Comment (joshmoore) [reply nightshade-webclients.yml]: Agreed. With nightshade it's more difficult, but I understood the general agreement to be:

 * Open a PR with checkmode against staging & prod server output
 * If general agreement, run against staging unmerged
 * If everything looks ok, merge and then rollout to prod

**Comment by sbesson** [comment]
@manics are you happy with the last commits. Should we get this merged in preparation of the OMERO 5.5.1 upgrade changes across prod servers?

**Pull request merged**